LiskHQ/lisk-nano
# Remove direct calls to methods of components in unit tests - Closes #703

### What was the problem?
Preferred is not to call any component method directly in unit tests. same goes for `setState` and other react related methods.

### How did I fix it?
I'm using enzyme event simulation to trigger all the required changes to test the methods of our components.

### How to test it?
Check the code syntax in unit tests. there should be no call for `setState`, `wrapper.instance` or any of the direct methods of the components.

### Review checklist
- [x] The PR solves #703 
- [x] None of our existing unit tests use `setState`, `wrapper.instance`.
